blog/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from blog.views import PostListView, PostDetailView
+urlpatterns = [
+    path("posts/", PostListView.as_view(), name="index"),
```

Comment:
The `urlpatterns` list is defined, but it seems to include a path to `blog.urls`, which might be incorrect if this file is intended to define the `blog` app's URLs. Make sure this file defines the paths specific to the `blog` application, such as the home page path with the name `index` as required by the task.